### DISCUSSION TITLE
Duo 2: How are esim managed?

### DISCUSSION THREAD

**Discussion by wokawoka** [Archfx/duo-de#53] [General]
I'm very much interested in buying a surface duo 2 and flashing this nice rom.
Could anyone confirm that esim are correctly managed?

I would also be interested in flashing the vanilla version (without google services), I assume that managing esims without google services would not be possible. Am I right?

Thank you!

**Comment by Ethanol10** [comment]
I can't seem to find any option to add eSIMs even with google services

**Comment by Archfx** [comment]
By default, there is no option for esim. However, you can use an [IMS app](https://source.android.com/docs/core/connect/ims) to get this feature. You can get help from the users who are using IMS (check the issues section to find).

**Reply by wokawoka** [reply]
> By default, there is no option for esim. However, you can use an [IMS app](https://source.android.com/docs/core/connect/ims) to get this feature. You can get help from the users who are using IMS (check the issues section to find).

do you mean [this issue](https://github.com/Archfx/duo-de/issues/30)?
Thank you!
